zerocopy-derive/tests/ui-nightly/privacy.rs

```diff
@@ -0,0 +1,102 @@
+// Copyright 2019 The Fuchsia Authors
+//
+// Licensed under a BSD-style license <LICENSE-BSD>, Apache License, Version 2.0
+// <LICENSE-APACHE or https://www.apache.org/licenses/LICENSE-2.0>, or the MIT
+// license <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your option.
+// This file may not be copied, modified, or distributed except according to
+// those terms.
+
+#![allow(unused_braces)]
+
+#[macro_use]
+extern crate zerocopy;
+
+fn main() {}
+
+// These tests cause errors which are generated by a later compilation pass than
+// the other errors we generate, and so if they're compiled in the same file,
+// the compiler will never get to that pass, and so we won't get the errors.
```

Comment:
That comment only makes sense in the `late_compile_pass.rs` test IMO. Is it true for this file too?